ios/swift/AmpliSwiftSampleApp/AmpliSwiftSampleApp.xcodeproj/xcshareddata/xcschemes/AmpliSwiftSampleApp (iOS).xcscheme

```diff
@@ -63,7 +63,7 @@
       <EnvironmentVariables>
          <EnvironmentVariable
             key = "AMPLITUDE_API_KEY"
-            value = ""
+            value = "AMPLITUDE_API_KEY"
```

Comment:
Having key and value be the same could be confusing. Can you please make this empty again.